apis/privilegedaccessmanager/v1alpha1/types.generated.go

```diff
@@ -137,6 +135,9 @@ type PrivilegedAccess struct {
 // about IAM.
 // +kcc:proto=google.cloud.privilegedaccessmanager.v1.PrivilegedAccess.GcpIamAccess
 type GcpIamAccess struct {
+	/* The following fields should be hidden because their values must match
```

Comment:
nit: Suggest listing the reasons for this decision so we can revisit when the condition changes (and I think it will): 
1. The current API has several redundant and duplicate fields between the parent URL and the IAMAccess. The usage is unclear.
2. It is cheaper to add a new API field than changing an existing one (and make it backward compatible)
3. We have pending discussions with PAM to better understand the API. But we don't want to be blocked right now.

Reply:
Done.